src/components/Temurin/NightlyTable/index.tsx

```diff
@@ -33,6 +34,7 @@ const TemurinNightlyTable = ({ results, openModalWithChecksum }: { results: { re
                   (asset, idx) => (
                     <tr key={release.release_name + key + asset.type + idx}>
                       <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-300">{capitalize(asset.os)}</td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-300">{capitalize(asset.architecture)}</td>
```

Comment:
No need to capitalise 

```suggestion
                      <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-300">{asset.architecture}</td>
```